src/objects/domain_obj.f90

```diff
@@ -50,8 +50,28 @@ module subroutine init(this, options)
 
         call setup_meta_data(this, options)
 
+        call set_var_lists(this, options)
     end subroutine
 
+    subroutine set_var_lists(this, options)
```

Comment:
The subroutine `setup_var` exists on line 587 or `domain_obj.f90`. Would it be appropriate to rename this subroutine to `setup_advect_vars` since it seems to deal specifically with the advection cases? Just to avoid a clash of similar names and follow a similar naming convention?

Reply:
Ok, I'm going to modify my comments a bit. After talking with Ethan, he pointed out that 'lists" is appropriate in the name because it is handling lists. We were thinking that maybe calling it `setup_advect_var_lists` would be appropriate. And if there are plans for more lists functions handling specific cases, that having a `setup_var_lists` subroutine that would call those other lists functions as well as `setup_advect_var_lists` work work well.

Reply:
Bingo -- thanks for the careful thought, I do have additional lists which are setup here (mainly for IO operations) which will be included in future PRs